index.html

```diff
@@ -0,0 +1,195 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>The Odin Project</title>
+    <link rel="stylesheet" href="style/style.css">
```

Comment:
put your main stylesheet after everything so that it can rewrite if it needs to :)